File: app/src/main/java/com/dmi/perfectreader/app/TimberReleaseTree.java
package com.dmi.perfectreader.app;

import android.util.Log;

import timber.log.Timber;

public class TimberReleaseTree extends Timber.DebugTree {
    @Override protected void log(int priority, String tag, String message, Throwable t) {
        if (priority != Log.VERBOSE && priority != Log.DEBUG) {
            super.log(priority, tag, message, t);
        }
    }
}

File: app/src/main/kotlin/com/dmi/perfectreader/app/TimberReleaseTree.kt
package com.dmi.perfectreader.app

import android.util.Log

import timber.log.Timber

class TimberReleaseTree : Timber.DebugTree() {
    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
        if (priority != Log.VERBOSE && priority != Log.DEBUG) {
            super.log(priority, tag, message, t)
        }
    }
}
